pkg/rpc/server/server.go

```diff
@@ -1088,11 +1096,23 @@ func (s *Server) invokeFunction(reqParams request.Params) (interface{}, *respons
 	if err != nil {
 		return nil, response.ErrInvalidParams
 	}
-	script, err := request.CreateFunctionInvocationScript(scriptHash, reqParams[1:])
+	var hashesForVerifying []util.Uint160
+	hashesForVerifyingIndex := len(reqParams)
+	if hashesForVerifyingIndex > 3 {
+		p := reqParams.ValueWithType(3, request.ArrayT)
+		if p != nil {
+			hashesForVerifying, err = p.GetArrayUint160FromHex()
+			if err != nil {
+				return nil, response.ErrInvalidParams
+			}
+		}
+		hashesForVerifyingIndex--
+	}
+	script, err := request.CreateFunctionInvocationScript(scriptHash, reqParams[1:hashesForVerifyingIndex])
```

Comment:
Is `hashesForVerifyingIndex` always 2 ?

Reply:
I considered two cases here:
1) `contractHash` + `callingMethod` + `methodArguments`
2) `contractHash` + `callingMethod` + `methodArguments` + `hashesForVerifying`

Am I missing something?